src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,23 @@
+import { jwtService } from '../services/jwt.service.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    res.status(401).send('Token is not valid');
```

Comment:
Consider providing a more descriptive error message or logging the error for debugging purposes. This can help in identifying issues during development.